### DISCUSSION TITLE
v3 / v4 로그인 관련 (정지 유무)

### DISCUSSION THREAD

**Discussion by ZZLOCO** [storycraft/node-kakao#616] [Q&A] [answered]
v3 이용시 버전이 맞지 않아, 정지 대상이 됩니다. (사용 불가능)
v4 정상적으로 사용가능합니다. 

v3 에서 모듈을 수정한다면, 정상적으로 사용 가능.

**Comment by tadashi74** [comment]
이용자 보호조치는 존버가 답이에요..ㅠ

**Comment by jipsoft1** [comment, marked as answer]
이보 걸렸다가 조금 기다렸더니 영정 먹던데요 전 ㅠㅠ

**Comment by ggoek34** [comment]
zzloco님 궁금한점이 있어서 그러는데 여쭤보려고하는데 혹시 텔레그램이나 카톡 아이디 알 수 있을까요?
고수님의 의견이 필요합니다 ㅠㅠ
